gradle/dependencies.gradle

```diff
@@ -16,9 +16,10 @@ dependencies {
   api('io.github.bonigarcia:webdrivermanager:3.7.1')
 
   implementation('com.browserup:browserup-proxy-core:2.0.1')
-  implementation('io.netty:netty-codec:4.1.42.Final')
+  implementation('io.netty:netty-all:4.1.44.Final')
+  implementation('io.netty:netty-codec:4.1.44.Final')
```

Comment:
Is `netty-all` contains `netty-codec` dependency?

Reply:
yes. Removed `netty-codec`.